### ISSUE TITLE
training speed?

### ISSUE THREAD

**Issue by pannaf** [ildoonet/tf-pose-estimation#496]
What's the training speed we should see for this? It took a really long time for it to display the epoch=0.26 step=500 lines after I started training.

> 2019-08-09 13:18:29.400500: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 22485 MB memory) -> physical GPU (device: 0, name: TITAN RTX, pci bus id: 0000:05:00.0, compute capability: 7.5)
> 2019-08-09 13:18:29.402157: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 22845 MB memory) -> physical GPU (device: 1, name: TITAN RTX, pci bus id: 0000:09:00.0, compute capability: 7.5)
> [2019-08-09 13:18:29,404] [train] [INFO] model weights initialization
> I0809 13:18:29.404209 140127765751616 train.py:171] model weights initialization
> 2019-08-09 13:18:29.609039: W tensorflow/core/common_runtime/colocation_graph.cc:1016] Failed to place the graph without changing the devices of some resources. Some of the operations (that had to be colocated with resource generating operations) are not supported on the resources' devices. Current candidate devices are [
>   /job:localhost/replica:0/task:0/device:CPU:0].
> See below for details of this colocation group:
> Colocation Debug Info:
> Colocation group had the following types and supported devices:
> Root Member(assigned_device_name_index_=-1 requested_device_name_='/device:GPU:*' assigned_device_name_='' resource_device_name_='/device:GPU:*' supported_device_types_=[CPU] possible_devices_=[]
> AssignAdd: CPU
> Const: GPU CPU XLA_CPU XLA_GPU
> Assign: CPU
> Identity: GPU CPU XLA_CPU XLA_GPU
> VariableV2: CPU
> 
> Colocation members, user-requested devices, and framework assigned devices, if any:
>   Variable (VariableV2) /device:GPU:*
>   Variable/Assign (Assign) /device:GPU:*
>   Variable/read (Identity) /device:GPU:*
>   Adam/value (Const) /device:GPU:*
>   Adam (AssignAdd) /device:GPU:*
>   save/Assign_420 (Assign) /device:GPU:*
> 
> [2019-08-09 13:18:31,520] [train] [INFO] Restore pretrained weights... ../models/numpy/openpose_coco.npy
> I0809 13:18:31.520933 140127765751616 train.py:181] Restore pretrained weights... ../models/numpy/openpose_coco.npy
> [2019-08-09 13:19:37,232] [train] [INFO] Restore pretrained weights...Done
> I0809 13:19:37.232814 140127765751616 train.py:192] Restore pretrained weights...Done
> [2019-08-09 13:19:37,233] [train] [INFO] prepare file writer
> I0809 13:19:37.233007 140127765751616 train.py:194] prepare file writer
> [2019-08-09 13:19:39,530] [train] [INFO] prepare coordinator
> I0809 13:19:39.530293 140127765751616 train.py:197] prepare coordinator
> [2019-08-09 13:19:39,664] [train] [INFO] Training Started.
> I0809 13:19:39.664165 140127765751616 train.py:202] Training Started.
> 2019-08-09 13:19:45.197568: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
> [2019-08-09 13:37:02,814] [train] [INFO] epoch=0.26 step=500, 30.6763 examples/sec lr=0.001000, loss=345.974, loss_ll=136.425, loss_ll_paf=92.2635, loss_ll_heat=44.1619
> I0809 13:37:02.814454 140127765751616 train.py:220] epoch=0.26 step=500, 30.6763 examples/sec lr=0.001000, loss=345.974, loss_ll=136.425, loss_ll_paf=92.2635, loss_ll_heat=44.1619

I can see it running on my GPU (98%):
`+-----------------------------------------------------------------------------+
| NVIDIA-SMI 430.40       Driver Version: 430.40       CUDA Version: 10.1     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  TITAN RTX           Off  | 00000000:05:00.0  On |                  N/A |
| 83%   86C    P2   213W / 280W |  23469MiB / 24217MiB |     98%      Default |
+-------------------------------+----------------------+----------------------+`
